applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, model, custom_settings):
         self.settings.ValidateAndAssignDefaults(default_settings)
 
         # Construct the linear solvers
-        import linear_solver_factory
+        from KratosMultiphysics import python_linear_solver_factory as linear_solver_factory
```

Comment:
this is already part of #3974 
I would prefer if you exclude it from this PR such that I don't have to solve conflicts many times.
Thx!

(and sorry for breaking your stuff)

Reply:
Okay, I will take them out. Or can you merge first? I will resolve the conflict.

Reply:
I revert back